ci/properties/go-ossf-slsa3-publish.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "SLSA Go releaser",
+    "creator": "Open Source Security Foundation (OpenSSF)",
+    "description": "Compile your Go project using a SLSA3 compliant builder. See https://github.com/slsa-framework/slsa-github-generator to learn more about the buidler, and slsa.dev to learn more about the benefits of SLSA",
```

Comment:
Too long description, will clutter the UI. Revert back to older one -> `Compile your Golang project using a SLSA3 compliant builder.`
Other details have been added at top of yml file so not needed here.

Reply:
Done, thank you